grails-web-url-mappings/src/test/groovy/org/grails/web/mapping/LinkGeneratorSpec.groovy

```diff
@@ -289,6 +290,29 @@ class LinkGeneratorSpec extends Specification {
             cacheKey == "somePrefix[resource:org.grails.web.mapping.Widget->2]"
     }
 
+    @Issue('https://github.com/grails/grails-core/issues/13627')
+    def 'resource links should use ident and allow controller override'() {
+        given:
+        final webRequest = GrailsWebMockUtil.bindMockWebRequest()
+        MockHttpServletRequest request = webRequest.currentRequest
+        linkParams.method = 'GET'
+
+        when: 'a resource is specified, ident() is used for id'
+        linkParams.resource = new Widget(id: 1, name: 'Some Widget')
+
+        then:
+        link == "/bar/widget/1/show"
```

Comment:
Related to https://github.com/grails/grails-core/pull/13628/files#diff-3e98c5bfca0e230c95b2e444b5345889eb2770d43cd5124003b3287efaee4d3eR368